content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -0,0 +1,108 @@
+# Generate live (real-time) metrics and plots for experiments
+
+When you
+[submit a new experiment](/doc/studio/user-guide/projects-and-experiments/run-experiments)
+from Iterative Studio, the
+[CI/CD setup](/doc/use-cases/ci-cd-for-machine-learning) in your Git repository
+gets invoked. If this setup includes a model training process, it will be
+triggered.
+
+In this model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This will enable you to view all intermediate results in Studio
+while your experiment is still running.
+
+This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-and-view-live-metrics-and-plots)
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics and plots
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+To set up the access token, open your user profile page. In the , click on
+`Generate new token`.
+
+You can generate, regenerate, or delete your access token from the
+`Studio access token` section of your profile page.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
+and the access token) at once. This is handy if you suspect that your account
+security may have been compromised.
+
+## Configure your model training CI job
+
+You should define the following environment variables in your CI job:
+
+- `STUDIO_REPO_URL`: URL for the Git repository connected to the project. E.g.,
+  `github:iterative/example-get-started`
+- `STUDIO_TOKEN`: Your access token. The access token must be present in any
+  request that sends data to the Iterative Studio ingestion endpoint. Requests
+  with missing or incorrect access tokens are rejected with an appropriate HTTP
+  error code and error message.
+
+One way to provide the above values to your CI job is to set them as GitHub
+secrets.
+
+Here is an example GitHub action.
+
+TODO: provide link to an example snippet (such as
+[this](https://github.com/iterative/test-dvclive-studio/blob/086a51d76c7983f24c091e1b007820916aa75e7d/.github/workflows/test_live_metrics.yaml#L17-L19))
+
+## Send and view live metrics and plots
+
+In the training CI job (which has been configured as detailed above), whenever
+you log your metrics or plots using [DVCLive], they will be automatically sent
+to Iterative Studio.
+
+Iterative Studio stores the live metrics and plots data in its database.
```

Comment:
💅🏼 found another p:
```suggestion
to Iterative Studio. Iterative Studio stores the live metrics and plots
data in its database.
```

Also, this answers my previous question but... do we need more info on what the Studio database is or how live metrics are stored? If it's not relevant to users (not something they can access directly or interact with), then prob not.

Reply:
Users can't access them directly.